pkgs/desktops/gnome-3/core/nautilus/default.nix

```diff
@@ -21,6 +21,7 @@ stdenv.mkDerivation rec {
       # Thumbnailers
       --prefix XDG_DATA_DIRS : "${gdk_pixbuf}/share"
       --prefix XDG_DATA_DIRS : "${librsvg}/share"
+      --prefix XDG_DATA_DIRS : "${shared_mime_info}/share"
```

Comment:
How about?

```
       --prefix XDG_DATA_DIRS : "${gdk_pixbuf}/share:${librsvg}/share:${shared_mime_info}/share"
```

Reply:
Or even `--prefix XDG_DATA_DIRS : "${stdenv.lib.makeSearchPath "share" [ gdk_pixbuf librsvg shared_mime_info ]}"`

Reply:
Thanks a lot for the tips. Changed :+1: